darts/models/torch_forecasting_model.py

```diff
@@ -205,6 +205,9 @@ def __init__(self,
         self.lr_scheduler_cls = lr_scheduler_cls
         self.lr_scheduler_kwargs = dict() if lr_scheduler_kwargs is None else lr_scheduler_kwargs
 
+        # by default models are block models (i.e. not recurrent)
+        self.is_recurrent = False
```

Comment:
Makes sense to create a method. But could we also make it a normal method that returns `False` and can optionally be overridden by subclasses? I'm just thinking in terms of reducing code because the block model is still the general case. But maybe that would not be ideal from a software engineering point of view?